Description: Remove build date from version string
 Using the C pre-processor __DATE__ macro prevents traceroute from being built
 reproducibly. In the context of Debian, adding the build date to the version
 string gives little information to users. So we just remove it.
Author: Jérémy Bobbio <lunar@debian.org>

Index: traceroute-2.0.20/traceroute/traceroute.c
===================================================================
--- traceroute-2.0.20.orig/traceroute/traceroute.c
+++ traceroute-2.0.20/traceroute/traceroute.c
@@ -72,7 +72,7 @@
 
 
 static char version_string[] = "Modern traceroute for Linux, "
-				"version " _TEXT(VERSION) ", " __DATE__
+				"version " _TEXT(VERSION)
 				"\nCopyright (c) 2008  Dmitry Butskoy, "
 				"  License: GPL v2 or any later";
 static int debug = 0;
